modules/boulder/CHANGELOG.md

```diff
@@ -0,0 +1,16 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## Unreleased
+
+### Changed
+- Replaced the FieldPresence and ValueProcessor ADTs with the native TypeScript equivalent #TINY-7549
```

Comment:
Should we include all the renames in here?

Reply:
Yeah, will do

Reply:
Think I got them all